### DISCUSSION TITLE
RPi2 processor not supported; can I build it myself?

### DISCUSSION THREAD

**Discussion by SniderThanYou** [haugene/docker-transmission-openvpn#2757] [Q&A]
Hey team,

It looks like docker images are not provided for the Raspberry Pi 2:

```sh
$ docker run ...
Unable to find image 'haugene/transmission-openvpn:latest' locally
latest: Pulling from haugene/transmission-openvpn
docker: no matching manifest for linux/arm/v6 in the manifest list entries.
```

I tried to build it myself, but the rabbit hole goes deeper:

```sh
$ git clone https://github.com/haugene/docker-transmission-openvpn.git
$ cd docker-transmission-openvpn.git
$ docker build .
Dockerfile:45
--------------------
  43 |           natpmpc
  44 |     
  45 | >>> FROM haugene/transmission-builder:4.0.5 as TransmissionBuilder
  46 |     
  47 |     FROM base
--------------------
ERROR: failed to solve: haugene/transmission-builder:4.0.5: no match for platform in manifest sha256:af52b34092c0deb33734918d5ab1dca5613a83079951c3ef6118c15180eedd93: not found
```

What are my options for running this on an arm v6 platform?

Thanks team!

**Comment by pkishino** [comment]
I’d start by figuring out if transmission can be built for it first

**Reply by SniderThanYou** [reply]
Thanks. I opted to just install transmission and openvpn natively, and set up the kill switch myself.

For anyone else trying to do this, I found success with this guide: https://jakobud.medium.com/automatic-anonymous-bittorrent-downloading-using-a-raspberry-pi-b367a67de238
